### PR TITLE
refactor: homepage page

### PR DESCRIPTION
**Pull request by moonlitgrace**
Fixing issues from previous commit: #70

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
+		<link rel="icon" href="%sveltekit.assets%/favicon.svg" />
```
Comment (baseplate-admin) [inline src/app.html:5]: Can we set web-component as icon?

File: src/lib/components/icons/core-isometric.svelte [hunk 2]
```diff
@@ -1,47 +1,74 @@
-<svg {...$$props} viewBox="0 0 480 680" fill="none" xmlns="http://www.w3.org/2000/svg">
-	<g id="Group_2" class="animate-float delay-0">
+<script lang="ts">
+	import type { SVGAttributes } from "svelte/elements";
+	const { ...props }: SVGAttributes<SVGElement> = $props();
+</script>
```
Comment (baseplate-admin) [inline src/lib/components/icons/core-isometric.svelte:4]: Uh nope. make it runes

File: src/lib/components/icons/core-isometric.svelte [hunk 3]
```diff
@@ -1,47 +1,74 @@
...
+	@for $i from 2 through 10 {
+		#Group_#{$i} {
+			animation-delay: (10 - $i) * 250ms;
+		}
+	}
+
+	g {
+		animation: float 3s ease-in-out infinite;
+	}
+</style>
```
Comment (baseplate-admin) [inline src/lib/components/icons/core-isometric.svelte:74]: maybe, you can use 
```
svg{
         @keyframes float {
		0%,
		100% {
			transform: translateY(0);
		}
		50% {
			transform: translateY(-10px);
		}
	}
	@for $i from 2 through 10 {
		#Group_#{$i} {
			animation-delay: (10 - $i) * 250ms;
		}
	}
	g {
		animation: float 3s ease-in-out infinite;
	}
};
```

File: src/lib/components/icons/coreicons.svelte [hunk 4]
```diff
@@ -1,4 +1,9 @@
-<svg {...$$props} viewBox="0 0 753 121" fill="none" xmlns="http://www.w3.org/2000/svg">
+<script lang="ts">
+	import type { SVGAttributes } from "svelte/elements";
+	const { ...props }: SVGAttributes<SVGElement> = $props();
+</script>
```
Comment (baseplate-admin) [inline src/lib/components/icons/coreicons.svelte:4]: Uh nope. make it runes

File: src/lib/components/icons/coreproject.svelte [hunk 5]
```diff
@@ -1,4 +1,9 @@
-<svg {...$$props} viewBox="0 0 849 150" fill="none" xmlns="http://www.w3.org/2000/svg">
+<script lang="ts">
+	import type { SVGAttributes } from "svelte/elements";
+	const { ...props }: SVGAttributes<SVGElement> = $props();
+</script>
```
Comment (baseplate-admin) [inline src/lib/components/icons/coreproject.svelte:4]: Uh nope. make it runes

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

**Comment by moonlitgrace** [comment]
Merging for now!

**Pull request merged**